lib/kino_db/connection_cell.ex

```diff
@@ -99,32 +113,65 @@ defmodule KinoDB.ConnectionCell do
     to_quoted(attrs, quote(do: MyXQL))
   end
 
+  defp to_quoted(%{"type" => "sqlite"} = attrs) do
```

Comment:
I think it will be simpler/shorter if just have a single `quote do ... end` piece for every type, even if there is some repetition. Maybe a separate function for the common postgrex/myxql options.